src/main_menu.cpp

```diff
@@ -136,6 +136,52 @@ std::vector<std::string> main_menu::load_file( const std::string &path,
     return result;
 }
 
+void main_menu::handle_input_timout( input_context &ctxt, std::string &action )
```

Comment:
I wonder why you "return" the action as out parameter instead of directly via `return action`? This makes the code that uses this function a bit confusing as one can not see that `action` is an output of the function (the call site does not show whether the argument is taken as const or non-const reference).

Also: why is it called `timout` - has it anything to do with a "Tim" going "out"?

Reply:
The evil copypasta god tricked me once more. Bless me oh great BevapDin and I shall repent my sinful deeds.

```bool game::handle_mouseview(input_context &ctxt, std::string &action)``` 

R.I.P Tim, you'll be missed.

PS: Passive aggressive much? I'd rather have your input on the error that occurs in only one build than whatever this BS is.